go/vt/vtgate/gateway.go

```diff
@@ -38,6 +37,7 @@ import (
 var (
 	// GatewayImplementation allows you to choose which gateway to use for vtgate routing. Defaults to tabletgateway, other option is discoverygateway
 	GatewayImplementation = flag.String("gateway_implementation", "tabletgateway", "Allowed values: discoverygateway (deprecated), tabletgateway (default)")
+	bufferImplementation  = flag.String("buffer_implementation", "healthcheck", "Allowed values: healthcheck (deprecated), keyspace_events (default)")
```

Comment:
Docstring says `keyspace_events` is the default but the actual default value has been specified as `healthcheck`.